identityprovider/orbitdb.go

```diff
@@ -57,7 +57,7 @@ func (p *OrbitDBIdentityProvider) SignIdentity(data []byte, id string) ([]byte,
 
 	signature, err := key.Sign(data)
 	if err != nil {
-		return nil, errors.New(fmt.Sprintf("Signing key for %s not found", id))
+		return nil, errmsg.KeyNotInKeystore
```

Comment:
```suggestion
		return nil, errmsg.KeyNotInKeystore.Wrap(err)
```

not sure if it was purposely omited (at least a second one in the same file)